src/validation.cpp

```diff
@@ -2810,6 +2810,18 @@ bool Chainstate::ConnectTip(BlockValidationState& state, CBlockIndex* pindexNew,
              Ticks<SecondsDouble>(time_total),
              Ticks<MillisecondsDouble>(time_total) / num_blocks_total);
 
+    // If we are the background validation chainstate, check to see if we are done
+    // validating the snapshot (i.e. our tip has reached the snapshot's base block).
+    if (this != &m_chainman.ActiveChainstate()) {
+        // This call may set `m_disabled`, which is referenced immediately afterwards in
+        // ActivateBestChain.
+        //
+        // If this fails (i.e. if snapshot is invalid), we will have marked the
+        // snapshot chainstate as unusable, and will have removed its coinsdb from
+        // disk.
```

Comment:
In commit "validation: add ChainMan logic for completing UTXO snapshot validation" (e17d51768bcf5f7791fdf7d0b2ee4b11cc64d922)

I think it's just renamed `_INVALID`, not removed. In any case, this detail doesn't really seem relevant here. It seems more relevant that the failure is handled internally so no need to check for errors from this function.